packages/workbox-precaching/controllers/PrecacheController.mjs

```diff
@@ -152,33 +153,33 @@ class PrecacheController {
       }
     }
 
-    const updatedEntries = [];
-    const notUpdatedEntries = [];
-
-    const cachePromises = [];
-    this._entriesToCacheMap.forEach((precacheEntry) => {
-      const promiseChain = this._cacheEntry(precacheEntry)
-      .then((wasUpdated) => {
-        if (wasUpdated) {
-          updatedEntries.push(precacheEntry);
-        } else {
-          notUpdatedEntries.push(precacheEntry);
-        }
-      });
+    const entriesToPrecache = [];
+    const entriesAlreadyPrecached = [];
 
-      cachePromises.push(promiseChain);
-    });
+    for (let precacheEntry of this._entriesToCacheMap.values()) {
```

Comment:
Nit: you can use `const` in for-of loops. Google style says to use `const` everywhere unless you have to use `let` due to reassignment.